### PR TITLE
Fix issue 71

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR fixes https://github.com/consensus-shipyard/ipc-solidity-actors/issues/71

### CODE REVIEW

File: src/lib/LibSubnetActorStorage.sol [hunk 1]
```diff
@@ -66,6 +64,7 @@ struct SubnetActorStorage {
     bytes32 currentSubnetHash;
     /// @notice Minimal number of validators required for the subnet to be able to validate new blocks.
     uint64 minValidators;
+    /// @notice Minimal number that a validator requires to stake to be added to the validator set of the subnet.
```
Comment (adlrocha) [inline src/lib/LibSubnetActorStorage.sol:67]:
```suggestion
    /// @notice Minimal collateral that a validator requires to stake to be added to the validator set of the subnet.
```

File: src/gateway/GatewayManagerFacet.sol [hunk 2]
```diff
@@ -68,6 +68,15 @@ contract GatewayManagerFacet is GatewayActorModifiers, ReentrancyGuard {
         }
 
         subnet.stake += msg.value;
+    }
+
+    /// @notice activateSubnet - activates the subnet if it has sufficient collateral
```
Comment (adlrocha) [inline src/gateway/GatewayManagerFacet.sol:73]: Isn't this redundant with the function above or am I missing something?

File: test/SubnetActorDiamond.t.sol [hunk 3]
```diff
@@ -695,7 +695,7 @@ contract SubnetActorDiamondTest is Test {
     }
 
     function callback() public view {
-        console.log("callback called");
+        // console.log("callback called");
```
Comment (adlrocha) [inline test/SubnetActorDiamond.t.sol:698]: Is this on purpose?

**Review by adlrocha — APPROVED** [review]
I am having a hard time understanding how this fix solves the issue described in #71. Do you mind describing the solution high-level? (we can sync if you think it may be easier)

File: test/SubnetActorDiamond.t.sol (hunk 3 above)
Comment (dnkolegov) [reply test/SubnetActorDiamond.t.sol:698]: image

Comment (dnkolegov) [reply test/SubnetActorDiamond.t.sol:698]: Just to disable this

**Pull request closed without merging**